client/src/Dashboard/Dashboard.jsx

```
@@ -0,0 +1,140 @@
+import React from 'react';
+import '../index.css';
+import { authenticationRepository, accountRepository } from '@/_services';
+import config from 'config';
+const signalR = require('@aspnet/signalr')
+
+class Dashboard extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            accountId: null,
+            currentPlan: null,
+            numberOfUsers: null,
+            maximumNumberOfUsersAllowed: null,
+            pricePerMonth: null,
+            hub: null
+        };
+
+        this.logout = this.logout.bind(this);
+        this.upgradePlan = this.upgradePlan.bind(this);
+    }
+
+    componentDidMount() {
+        const hub = this.configureUserHub();
+        this.start(hub);
```

Looks like server errors are ignored